cinema/urls.py

```diff
@@ -1,10 +1,43 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    MovieViewSet,
+    CinemaHallViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+
+cinema_hall_list = CinemaHallViewSet.as_view(
+    actions={"get": "list", "post": "create"}
+)
+cinema_hall_detail = CinemaHallViewSet.as_view(
+    actions={
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy"
+    }
```

Comment:
you should use routers here

Reply:
![Uploading ha.gif…]()

Reply:
![hz](https://github.com/mate-academy/py-api-views/assets/106765167/405b22d1-ce22-474a-8b46-fa71a21ae7df)